sdk/policyinsights/Azure.ResourceManager.PolicyInsights/api/Azure.ResourceManager.PolicyInsights.netstandard2.0.cs

```diff
@@ -56,10 +56,16 @@ public static partial class PolicyInsightsExtensions
         public static Azure.ResourceManager.PolicyInsights.PolicyAttestationResource GetPolicyAttestationResource(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier id) { throw null; }
         public static Azure.ResourceManager.PolicyInsights.PolicyAttestationCollection GetPolicyAttestations(this Azure.ResourceManager.ArmClient client, Azure.Core.ResourceIdentifier scope) { throw null; }
         public static Azure.Pageable<Azure.ResourceManager.PolicyInsights.Models.PolicyEvent> GetPolicyEventQueryResults(this Azure.ResourceManager.ManagementGroups.ManagementGroupResource managementGroupResource, Azure.ResourceManager.PolicyInsights.Models.PolicyEventType policyEventsResource, Azure.ResourceManager.PolicyInsights.Models.PolicyQuerySettings queryOptions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static Azure.Pageable<Azure.ResourceManager.PolicyInsights.Models.PolicyEvent> GetPolicyEventQueryResults(this Azure.ResourceManager.Resources.PolicyAssignmentResource policyAssignmentResource, Azure.ResourceManager.PolicyInsights.Models.PolicyEventType policyEventsResource, Azure.ResourceManager.PolicyInsights.Models.PolicyQuerySettings queryOptions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Could we also change the two parameter names here `policyEventsResource` and `queryOptions`? Their names do not quite match their purpose.
We could do this using the rename-rules temporarily.
I am introducing a new configuration to provide us the ability of renaming a parameter like the rename-mapping. I could change the temporary configuration to that once that is ready

Reply:
Renamed